encord/objects/ontology_labels_impl.py

```diff
@@ -450,6 +457,17 @@ def _batch_save_rows_reducer(
         bundle_payload.payload += payload.payload
         return bundle_payload
 
+    @property
+    def metadata(self) -> Optional[DICOMSeriesMetadata]:
```

Comment:
we have all sorts of metadata, like video client metadata, video metadata, etc. 
Can we give this a name more descriptive to "DICOM"?

Reply:
Ah I see what you're trying to do ... having the option to have multiple different return types here. I guess up to you if you want to anticipate of having some sort of `union` return type here, or instead just have multiple differently named metadatas that are all optional. What do you think?